frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -614,9 +614,12 @@ def generate_build_config(cls, coprdir, chroot_id):
             repos[0]["module_hotfixes"] = True
 
         if not copr.auto_createrepo:
+            # On backend we use /devel but in Pulp we cannot create subdirectories
+            suffix = "-devel" if copr.storage == StorageEnum.pulp else "/devel"
+            baseurl = copr.repo_url + "/{0}{1}/".format(chroot_id, suffix)
```

Comment:
Long term we should have utility function for this, like `copr_dir.repo_url(chroot_id, devel=False)`.
Seems like we intentionally ignore copr-dirs?